cmdb/framework/cmdb_log_manager.py

```diff
@@ -25,26 +27,45 @@
 class CmdbLogManager(CmdbManagerBase):
 
     def __init__(self, database_manager=None):
+        self._type_manager = TypeManager(database_manager)
         super(CmdbLogManager, self).__init__(database_manager)
 
     def search(self):
         pass
 
     # CRUD functions
-    def get_log(self, public_id: int):
+    def get_log(self, public_id: int, group_id: int):
```

Comment:
ACLs should always be optional!
A validation of the acls should only applied if a user AND a permission is set.
Look at the reference [implementaion](https://github.com/NETHINKS/DATAGERRY/blob/334d4e8ccb2e91c8c987a01c85dd227275ebe7eb/cmdb/framework/cmdb_object_manager.py#L120) for more details.